set.mm

```diff
@@ -102,7 +102,7 @@ been continuously enriched since then (list of contributors below). $)
 
 $( (See 'help verify markup' for help with modularization tags.) $)
 $( Begin $[ set-header.mm $] $)
-$(
+$( !
```

Comment:
The purpose of this is to break up the header comment sequence so that the comment is not interpreted as a regular header comment, because header comments have verified markup and the head comment has a lot of ascii syntax that does not conform to the markup parser's expectations (not least of which is the existence of multiple sub-headers).

Alternatives:
* Use another character like `.`
* Use something like `<not-a-header>`, which I wanted to avoid since readers will be confused by it since this is right at the start of the file
* Use a variation on the usual header decoration like `=~=~=~=~=~=~` so that it is not recognized as a header

Reply:
or maybe something like:
```
$( (text on this line to avoid comment being interpreded as a header comment)
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
```
?

Reply:
I would prefer the variation on the usual header decoration like =~=~=~=~=~=~ so that it is not recognized as a header.

Reply:
would the variant header decoration be used on all the headers in this comment? It wouldn't match with the rest of the file stylistically, which seems not great either.

Reply:
To be consequent - yes. Maybe a different style could be used for the 2nd level, e.g.,

-~-~-~-~-~-~-~-~-~-~-~-~
or
.~.~.~.~.~.~.~.~.~.~.~.~.~.~